### PR TITLE
feature/96_alarm_script

### PR DESCRIPTION
**Pull request by frbattid**
* Implements issue #96 
* Tested in FIWARE Lab
* Assignee @fgalan

### CODE REVIEW

File: cosmos-admin/README.md [hunk 1]
```diff
@@ -35,6 +36,30 @@ If for any reason the data copying is interrupted (for instance, the communicati
...
+This script has been designed for detecting certain scenarios the Cosmos administrator must be warned about. Specifically:
+
+* When a HDFS space is close to the quota limit. How much close it is depends on a configurable threshold.
+* When an account has no data within the HDFS space and certain time has last after the account creation. The interval check depends on a configurable value measured in days.
+
+When any of the above situations is detected, an email is sent to the Cosmos administrator.
+
+`warnings.sh` is parameterized by:
+
+*  Host running the MySQL server
```
Comment (fgalan) [inline cosmos-admin/README.md]: Indent

File: cosmos-admin/README.md [hunk 2]
```diff
@@ -35,6 +36,30 @@ If for any reason the data copying is interrupted (for instance, the communicati
...
+* Port where the MySQL server listens for requests. Typically, TCP/3306 port.
+* Database name. Usually, `cosmos`.
+* MySQL user allowed to insert data within the `cosmos` database, `cosmos_user` table.
+* Pasword for the MySQL user.
+* Percentage of HDFS space considered close to the quota limit.
+* Number of days since creation an account having no HDFS data is considered unused.
+* Email address (owned by an administrator) which sending the report to.
```
Comment (fgalan) [inline cosmos-admin/README.md]: to send the report to?

Comment (frbattid) [inline cosmos-admin/README.md]: Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/b770ceabb195f8f79891fca3c3b62603de778da3

File: cosmos-admin/README.md (hunk 1 above)
Comment (frbattid) [inline cosmos-admin/README.md]: Fixed in https://github.com/telefonicaid/fiware-cosmos/commit/b770ceabb195f8f79891fca3c3b62603de778da3

**Comment by fgalan** [comment]
LGTM

**Pull request merged**